src/Umbraco.Tests.Integration/Cache/DistributedCacheBinderTests.cs

```diff
@@ -47,9 +47,6 @@ public void Can_Find_All_Event_Handlers()
                 new EventDefinition<IUserService, SaveEventArgs<UserGroupWithUsers>>(null, UserService, new SaveEventArgs<UserGroupWithUsers>(Enumerable.Empty<UserGroupWithUsers>())),
                 new EventDefinition<IUserService, DeleteEventArgs<IUserGroup>>(null, UserService, new DeleteEventArgs<IUserGroup>(Enumerable.Empty<IUserGroup>())),
 
-                new EventDefinition<IDataTypeService, SaveEventArgs<IDataType>>(null, DataTypeService, new SaveEventArgs<IDataType>(Enumerable.Empty<IDataType>())),
```

Comment:
@bergmania I removed this to get test passing, but as the EventArgs for this is no longer a thing. Did I need to replace this with a different approach instead?

Reply:
Not Bjarke, but I had to do the same when I migrated Localization events, as far as I can tell we don't have a different approach for this yet.